localgov.install

```diff
@@ -80,9 +87,25 @@ function localgov_update_9502() {
 }
 
 /**
- * Enable msqrole module.
+ * Update existing sites to use entity_usage for media.
  */
```

Comment:
I'm a bit confused as to what's happened here?

This is still the MR for enabling some masquerade system?

Reply:
@ekes I think it was just the merge confict as the Update existing sites to use entity_usage for media PR went in first and they both originally used the same update hook number 9503.

Reply:
Actual change (move this to update 9504:  https://github.com/localgovdrupal/localgov/blob/33c491c3e366d34ed6ae0ac0201b808c863b6deb/localgov.install#L89-L112